storage/calculate-container-size/calculate-container-sizes-in-account.ps1

```diff
@@ -0,0 +1,105 @@
+# This script will show how to get the total size of the blobs in all containers in a storage account.
+# Before running this, you need to create a storage account, at least one container,
+#    and upload some blobs into that container.
+# note: this retrieves all of the blobs in each container in one command.
+#       Run the Connect-AzAccount cmdlet to connect to Azure.
+#       Requests that are sent as part of this tool will incur transactional costs.
+#
+
+$containerstats = @()
+
+# Provide the name of your storage account and resource group
+$storage_account_name = "<name-of-your-storage-account>"
+$resource_group = "<name-of-your-resource-group"
+
+# Get a reference to the storage account and the context.
+$storageAccount = Get-AzStorageAccount `
+  -ResourceGroupName $resource_group `
+  -Name $storage_account_name
+$Ctx = $storageAccount.Context
+
+$container_continuation_token = $null
+do {
+  $containers = Get-AzStorageContainer -Context $Ctx -MaxCount 5000 -ContinuationToken $container_continuation_token    
+  $container_continuation_token = $null;
+
+  if ($containers -ne $null)
+  {
```

Comment:
```suggestion
  if ($containers -ne $null) {
```
Use OTBS (One true brace style) so your `if` statements are formatted the same as the code for the `do` loops.